make-release.sh

```diff
@@ -38,6 +38,9 @@ sed -i -e "s/cloudshell-image:v\([0-9\.]\+\)/cloudshell-image:${NEW_VERSION}/g"
 sed -i -e "s/cloudshell_git_branch=v\([0-9\.]\+\)/cloudshell_git_branch=${NEW_VERSION}/g" ${REPO_ROOT}/docs/index.html;
 sed -i -e "s/cloudshell-image:v\([0-9\.]\+\)/cloudshell-image:${NEW_VERSION}/g" ${REPO_ROOT}/docs/index.html;
 
+# update tags for load generator image
+find "${REPO_ROOT}/kubernetes-config" -name '*.yaml' -exec sed -i -e "s/basic:[0-9]+\.[0-9]+\.[0-9]+/basic:${NEW_VERSION}/g" {};
```

Comment:
This seems to be doing a find and repace on a version code, when it should be replacing latest
You'll also need to add a line to revert back to latest later, and to run git add on the file

You can run `./make-release --dry-run` and  `./make-release --no-push` to make sure everything works as expected